### PR TITLE
Add present validator

### PR DESCRIPTION
**Pull request by keito-jp**
see #16

### CODE REVIEW

File: lib/validators/present.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  constructor(public definition: IPresentValidatorDefinition) {}
+
+  public validate(input: any): IValidationError<any, IPresentValidatorDefinition> {
+    const invalid = {
+      definition: this.definition,
+      input,
+    };
+    if (
+      input == null ||
+      input === "" ||
```
Comment (minodisk) [inline lib/validators/present.ts]: `""` is checked in `rWhiteSpce`, isn't it?

File: lib/validators/present.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      PresentValidator.rWhiteSpace.test(input)
+    ) {
+      return invalid;
+    }
+    if (underscore.isArray(input) || underscore.isObject(input)) {
+      if (underscore.isEmpty(input)) {
```
Comment (minodisk) [inline lib/validators/present.ts]: Don't nest simple `if`. Make 2 conditions to 1 condition.

**Comment by keito-jp** [comment]
Fixed!

File: package.json [hunk 3]
```diff
@@ -31,5 +31,8 @@
     "typescript-formatter": "^2.3.0",
     "typescript-simple": "git://github.com/uncovertruth/typescript-simple.git#fix/file-text",
     "typings": "^1.3.2"
+  },
+  "dependencies": {
+    "underscore": "^1.8.3"
```
Comment (minodisk) [inline package.json]: Use `lodash` module.
Install these modules:
- `lodash.isstring`
- `lodash.isarray`
- `lodash.isobject`

**Comment by keito-jp** [comment]
@minodisk fixed!

**Pull request merged**